Livecheckables/gtksourceview.rb

```diff
@@ -0,0 +1,4 @@
+class Gtksourceview
+  livecheck :url   => "https://download.gnome.org/sources/gtksourceview/",
+            :regex => /gtksourceview-(2\.([0-8]\d*?)?[02468](?:\.\d+)+)\.t/
```

Comment:
```suggestion
            :regex => /gtksourceview-(2\.([0-8]\d*?)?[02468](?:\.\d+)*?)\.t/
```

Bringing the regex in line with the `download.gnome.org` strategy's regex.